src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: { categories: [], status: 'Under construction' },
```

Comment:
- [ ] The category initialState should contain an empty array, kindly consider implementing this